### PR TITLE
Normalize common names

### LINKED ISSUE #23: Make capitalization of common names regular
All lowercase except for proper nouns.

Also clean up long names that contain "or".

### PR DESCRIPTION
**Pull request by helloworld12321**
This commit makes all of the common names lower case, except if the name contains a proper noun (for example, "Spotted Joe Pye Weed" → "spotted Joe Pye weed").

In cases where something had two alternate names, I picked the one that seemed best (for example, "wild bergamot or bee balm" → "wild bergamot"). (For an explanation of why I picked the names I did, see the commit message for cbc66e7.)

I'm not dealing with hyphenation and compound words in this pull request, since that's significantly more subjective; I think mostly fine if we just go with what we have now.

(This pull request also updates the schemas a bit, so that they make stronger guarrantees about the structure of the data.)

Closes #23.

### CODE REVIEW

**Comment by floogulinc** [comment]
The regex pattern for scientific names is neat.

File: spec/bees.spec.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ const {nests} = require('../nests.json');
 describe('The bee JSON data', () => {
   for (const [id, bee] of Object.entries(bees)) {
     describe(id, () => {
+      it('Has only one common name, not two common names separated by "or"', () => {
+        expect(bee.name).not.toMatch(/\bor\b/);
```
Comment (floogulinc) [inline spec/bees.spec.js:9]: Can't this be done in the schema?

Comment (helloworld12321) [reply spec/bees.spec.js:9]: Oh, I suppose, thanks!

Comment (helloworld12321) [reply spec/bees.spec.js:9]: Hrm&hellip; on second thought, I'd prefer to not use the schema for that—negative regular expressions, like "does not contain the world 'or'", can get hard to write and read. Using `.not.toMatch()` is a lot nicer, even if Jasmine is a bit overkill job :slightly_smiling_face:

File: bees.schema.json [hunk 2]
```diff
@@ -14,21 +14,27 @@
...
           },
           "sci_name": {
+            "description": "This flower's scientific name.",
             "type": "string",
-            "pattern": "^[A-Z][a-zäëïöüÿ\\-]* [a-z][a-zäëïöüÿ\\-]+$"
+            "pattern": "^[A-Z][a-zäëïöüÿ\\-]* [a-z][a-zäëïöüÿ\\-]+$",
+            "$comment": "See https://www.iapt-taxon.org/nomen/pages/main/art_60.html for the precise structure of scientific names."
           },
           "relative_size": {
+            "description": "Roughly how large this bee is compared to an average bee. 1 is normal, 0.1 is small, and 10 is big.",
```
Comment (floogulinc) [inline bees.schema.json:31]: I think in practice the range of reasonable sizes is more like 0.1-2 and even that is probably more range than is actually used. We probably want to bring down the maximum here.

**Review by floogulinc — CHANGES_REQUESTED** [review]
See my previous comment, it's pretty minor.

**Comment by helloworld12321** [comment]
All done—thanks! :)

**Review by floogulinc — APPROVED** [review]

**Pull request merged**